src/infra/auth-rebac/inmem/tests/repos/test_inmem_rebac_repository.rs

```diff
@@ -7,91 +7,81 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
+use database_common_macros::database_transactional_test;
 use dill::{Catalog, CatalogBuilder};
 use kamu_auth_rebac_inmem::InMemoryRebacRepository;
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
-#[test_log::test(tokio::test)]
-async fn test_try_get_properties_from_nonexistent_entity() {
-    let harness = InmemRebacRepositoryHarness::new();
-
-    kamu_auth_rebac_repo_tests::test_try_get_properties_from_nonexistent_entity(&harness.catalog)
-        .await;
-}
+database_transactional_test!(
```

Comment:
Let's use it everywhere, not just this domain.

Reply:
Will be done in a separate PR